src/components/Nav.js

```diff
@@ -0,0 +1,27 @@
+import { Link } from "react-router-dom";
+
+const Nav = () => {
+  return (
```

Comment:
Suggestion: would be cool, at some point, to have the category nav links render programmatically, to account for new categories being added in future